db/seeds.rb

```diff
@@ -2,47 +2,57 @@
 puts "CREATED ADMIN USER: " << admin.email
 
 # Add sites
-sites = [{
-  url: "https://atomicjolt.instructure.com",
-  oauth_key: Rails.application.secrets.canvas_developer_id,
-  oauth_secret: Rails.application.secrets.canvas_developer_key
-}]
+sites = [
+  {
+    url: Rails.application.secrets.canvas_url,
+    oauth_key: Rails.application.secrets.canvas_developer_id,
+    oauth_secret: Rails.application.secrets.canvas_developer_key
+  },
+]
 
 # Add an LTI Application
-applications = [{
-  name: "LTI Admin",
-  description: "LTI tool administration",
-  client_application_name: "lti_admin_app",
-  canvas_api_permissions: "",
-  kind: Application.kinds[:admin],
-}, {
-  name: "LTI Starter App",
-  description: "LTI Starter App by Atomic Jolt",
-  client_application_name: "app",
-  # List Canvas API methods the app is allowed to use. A full list of constants can be found in canvas_urls
-  canvas_api_permissions: "LIST_ACCOUNTS",
-}]
+applications = [
+  {
+    name: "LTI Admin",
+    description: "LTI tool administration",
+    client_application_name: "lti_admin_app",
+    canvas_api_permissions: "",
+    kind: Application.kinds[:admin],
+  },
+  {
+    name: "LTI Starter App",
+    description: "LTI Starter App by Atomic Jolt",
+    client_application_name: "app",
+    # List Canvas API methods the app is allowed to use. A full list of constants can be found in canvas_urls
+    canvas_api_permissions: "LIST_ACCOUNTS",
+  },
+]
 
-application_instances = [{
-  application: "LTI Admin",
-  lti_key: "lti-admin",
-  url: "https://atomicjolt.instructure.com",
-  domain: "admin.#{ENV['APP_URL']}"
-}, {
-  application: "LTI Starter App",
-  lti_key: Rails.application.secrets.default_lti_key,
-  lti_secret: Rails.application.secrets.default_lti_secret,
-  url: "https://atomicjolt.instructure.com",
-  # This is only required if the app needs API access and doesn't want each user to do the oauth dance
-  canvas_token: Rails.application.secrets.canvas_token,
-  # Each application instance can have it's own custom domain. Typically, this is not needed
-  # as the application will use the oauth_consumer_key from the LTI launch to partition different
-  # application instances. However, if Canvas is launching the LTI tool based on url then you will
-  # need a different domain for that tool since Canvas uses the domain to find the LTI tool among
-  # all installed LTI tools. If two tools share the same domain then the tool discovered by Canvas
-  # to do the LTI launch will be indeterminate
-  domain: "#{Rails.application.secrets.default_lti_key}.#{ENV['APP_URL']}"
-}]
+application_instances = [
+  {
+    application: "LTI Admin",
+    tenant: "lti-admin",
+    lti_key: "lti-admin",
+    url: Rails.application.secrets.canvas_url,
+    domain: "admin.#{ENV['APP_URL']}"
```

Comment:
Should we be mixing rails.application.secrets and ENV vars? Maybe just one or the other?